tests/unit/simple_mock_invokemethod_v1.py

```diff
@@ -1,23 +1,26 @@
 """
-mock_pywbem test script that installs a a method provider for the class
-CIM_Foo
-"""
+Test mock script that installs a test method provider for CIM method
```

Comment:
Suggestion on naming of the python files for this and the following.  Change the last prefix to aimplw_moxk_invokwmwrho_v1_old and ..._new since "-s" does not have a lot of meaning.

Reply:
DONE. Renamed as suggested.